tdrs-backend/tdpservice/parsers/schema_defs/header.py

```diff
@@ -13,34 +13,34 @@
     ],
     postparsing_validators=[],
     fields=[
-        Field(name='title', type='string', startIndex=0, endIndex=6, required=True, validators=[
+        Field(item=1, name='title', type='string', startIndex=0, endIndex=6, required=True, validators=[
             validators.matches('HEADER'),
```

Comment:
It seems like we already have a unique value for name. Is there a reason we needed to add unique item numbers?

Reply:
it was added solely to populate the `ParserError` object and include it in the downloaded report. i think there was some early ux feedback suggesting it wasn't useful though so i'd be open to removing it

Reply:
Yeah, it might be frustrating to have a name available, but spit out `'item 7' field failed parsing` instead of something more meaningful to us like `'program_type' field failed parsing requirements`. I'm sure we could include both, maybe it's useful to know approximately where on the page to look (7th field) and a meaningful name for that field. I'm mostly okay with leaving it in, just seemed redundant to me.

Reply:
@jtimpe @reitermb can you remind me how we are using `item` in the context of these errors? historically, this is a reference to the data elements in the [instructions](https://www.acf.hhs.gov/sites/default/files/documents/ofa/tanf_data_reports_tan_ssp_instructions_definitions.pdf) and are mentioned in system error reports like [this](https://www.acf.hhs.gov/sites/default/files/documents/ofa/tanf_fatal_edits_sections_1_and_4.pdf) 

are we planning to just mention the data element label (which our STTs would be most familiar with)? or perhaps both (e.g. `ITEM 4 (RPT_MONTH_YEAR)`)

Reply:
at this point, it's only used to populate a column in the error report spreadsheet. i can include it in the field validation messaging as well if that's what we want

Reply:
i think this is something we'll discuss at ux sync next week -- i need a refresher on what we learned about this in the UX research :)

Reply:
To be resolved [here](https://github.com/raft-tech/TANF-app/issues/2564) per Dev sync 6/7